### PR TITLE
CLOUDP-201110: Update errors

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

[CLOUDP-201110](https://jira.mongodb.org/browse/CLOUDP-201110)

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — DISMISSED** [review]

File: internal/cli/atlas/deployments/setup.go [hunk 1]
```diff
@@ -69,15 +69,15 @@ const (
...
-	errCancel                       = errors.New("setup cancelled")
-	errMustBeInt                    = errors.New("input must be an integer")
-	errPortOutOfRange               = errors.New("port must within the range 1..65535")
-	errPortNotAvailable             = errors.New("port not available")
-	errFlagTypeRequired             = errors.New("flag --type is required when --force is set")
-	errInvalidDeploymentType        = errors.New("invalid deployment type")
-	errInvalidMongoDBVersion        = errors.New("invalid mongodb version")
-	errUnsupportedConnectWith       = errors.New("flag --connectWith unsupported")
-	errDeploymentTypeNotImplemented = errors.New("deployment type not implemented")
+	errCancel                       = errors.New("the was setup cancelled")
```
Comment (andreaangiolillo) [inline internal/cli/atlas/deployments/setup.go:72]: 👀 This may need an update

Comment (corryroot) [inline internal/cli/atlas/deployments/setup.go:72]:
```suggestion
	errCancel                       = errors.New("the setup was cancelled")
```

File: internal/cli/atlas/deployments/deployments.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ import (
 )
 
 var (
-	errCompassNotInstalled = errors.New("MongoDB Compass not found in your system, to install MongoDB Compass follow these instructions: https://www.mongodb.com/docs/compass/current/install/")
-	errMongoshNotInstalled = errors.New("mongosh not found in your system, to install MongoDB Shell follow these instructions: https://www.mongodb.com/docs/mongodb-shell/install")
+	errCompassNotInstalled = errors.New("could not find MongoDB Compass, install MongoDB Compass: https://dochub.mongodb.org/core/install-compass")
```
Comment (corryroot) [inline internal/cli/atlas/deployments/deployments.go:26]:
```suggestion
	errCompassNotInstalled = errors.New("did not find MongoDB Compass, install: https://dochub.mongodb.org/core/install-compass")
```

File: internal/cli/atlas/deployments/deployments.go [hunk 3]
```diff
@@ -23,8 +23,8 @@ import (
...
+	errMongoshNotInstalled = errors.New("mongosh was not found, install mongosh: https://dochub.mongodb.org/core/install-mongosh")
```
Comment (corryroot) [inline internal/cli/atlas/deployments/deployments.go:27]:
```suggestion
	errMongoshNotInstalled = errors.New("did not find mongosh, install: https://dochub.mongodb.org/core/install-mongosh")
```

**Review by corryroot — COMMENTED** [review]
Commented with a few nits.

**Review by corryroot — APPROVED** [review]
LGTM

**Pull request merged**